manager/manager.go

```diff
@@ -726,6 +726,9 @@ func (m *Manager) rotateRootCAKEK(ctx context.Context, clusterID string) error {
 	if strPassphrase == "" {
 		return nil
 	}
```

Comment:
Oh that's a good point, and way for users to stop doing that themselves.  Thanks!